com.sayonara_player.Sayonara.json

```diff
@@ -0,0 +1,188 @@
+{
+    "app-id": "com.sayonara_player.Sayonara",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "sayonara",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--filesystem=host",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.mpris.MediaPlayer2.Player",
+        "--own-name=org.mpris.MediaPlayer2.sayonara",
+        "--own-name=org.kde.*",
+        "--device=all",
+        "--filesystem=xdg-run/gvfs"
+    ],
+    "rename-appdata-file": "sayonara.appdata.xml",
+    "rename-desktop-file": "com.sayonara-player.Sayonara.desktop",
+    "rename-icon": "sayonara",
+    "modules": [
+        {
+            "config-opts": [],
+            "buildsystem": "autotools",
+            "name": "soundtouch",
+            "sources": [
+                {
+                    "commit": "762f56024b7ade81f6565903161dffec0ad4741e",
+                    "tag": "2.2",
+                    "type": "git",
+                    "url": "https://gitlab.com/soundtouch/soundtouch.git"
+                }
+            ]
+        },
+        {
+            "config-opts": [
+                "--buildtype=release",
+                "-Ddoc=disabled",
+                "-Dexamples=disabled",
+                "-Dtests=disabled",
+                "-Dbenchmarks=disabled",
+                "-Dnls=disabled",
+                "-Dgobject-cast-checks=disabled",
+                "-Dglib-asserts=disabled",
+                "-Dglib-checks=disabled",
+                "-Dextra-checks=disabled"
+            ],
+            "buildsystem": "meson",
+            "name": "gstreamer",
+            "sources": [
+                {
+                    "commit": "a42fe476d3ee5576921f67a331464065ec33b9a4",
+                    "tag": "1.18.3",
```

Comment:
Is it here because the app needs newer gstreamer than available in the runtime?

Reply:
@barthalion : The app needs the pitch filter from soundtouch plugin in the package GStreamer Bad Plug-ins.

I think, that it is not available in the runtime (I let the bot build a new version without the gstreamer stuff) - and the app complained that 'pitch' could not created. 

Therefore I added the soundtouch and (latest) gstreamer modules to the manifest. I was unable to build just the Bad Plugins, so I've added the whole gstreamer stuff.

Do you recommend an other solution?

Reply:
No, it's fine, just wanted to make sure.